crates/holochain/src/conductor/conductor.rs

```diff
@@ -1079,6 +1079,13 @@ mod app_impls {
 
             let cells_to_create = ops.cells_to_create();
 
+            if cells_to_create
+                .iter()
+                .any(|(cell_id, _)| self.cell_by_id(cell_id).is_ok())
```

Comment:
Rather than using `cell_by_id` which only gets running cells, I think you should collect all cell ids in all apps via something like `ConductorState.installed_apps.values().flat_map(|app| app.all_cells())` https://github.com/holochain/holochain/blob/bf242f00f7ef84cd7f09efc9770dc632f0da4310/crates/holochain_types/src/app.rs#L498-L504